src/components/src/VRProductDisplayItem/README.md

```diff
@@ -12,6 +12,12 @@ For an example of how to integrate this component into your own project checkout
 
 `productdisplayitem.main.tsx`
 
+The component gives the user the ability to rotate in the vr-scene with mobile, dedicated vr devices and on desktop.  Look at the aframe website for a more exhaustive list.  
```

Comment:
We should provide a link to the A-Frame website here if we're referencing it.

Reply:
Sounds good

Reply:
There is a reference above in the README.  Ill reword it to reference it more clearly